dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/io/DataReaderImpl.java

```diff
@@ -130,7 +132,9 @@ private Unmarshaller createUnmarshaller() {
             }*/
             // Liberty change end
             um.setSchema(schema);
+            LOG.fine("Schema is set to Unmarshaller"); // Liberty change
             um.setAttachmentUnmarshaller(getAttachmentUnmarshaller());
+            LOG.fine("AttachmentUnmarshaller is set to Unmarshaller"); // Liberty change
```

Comment:
I'd like you to remove this and the above trace. I don't think it provides any useful information that would help debug future issues, and just congests the trace we already have. 

```suggestion
```